packages/image/src/resolve.js

```diff
@@ -144,12 +144,17 @@ const getImageFormat = body => {
 };
 
 const resolveImageFromUrl = async src => {
-  const { uri, body, headers, method = 'GET' } = src;
+  const { uri, body, headers, method = 'GET', credentials } = src;
 
   const data =
     !BROWSER && getAbsoluteLocalPath(uri)
       ? await fetchLocalFile(uri)
-      : await fetchRemoteFile(uri, { body, headers, method });
+      : await fetchRemoteFile(uri, {
+          body,
+          headers,
+          method,
+          ...(credentials && { credentials }),
```

Comment:
Can this construction be just a `credentials`? is empty key break fetch default?

```suggestion
          credentials,
```